test/fixtures.js

```diff
@@ -1,40 +1,86 @@
-module.exports = [{
+module.exports = [
+  {
     fakerMethod: 'address',
     output: ['country', 'city', 'province']
-},{
+  },
+  {
+    fakerMethod: 'animal',
+    output: ['bear', 'fish', 'butterfly']
+  },
+  {
+    fakerMethod: 'color',
+    output: ['human', 'rgb', 'blue']
+  },
+  {
+    fakerMethod: 'commerce',
+    output: ['department', 'productName', 'employee']
+  },
+  {
     fakerMethod: 'company',
     output: ['bsNoun', 'bsBuzz', 'startups']
-},{
+  },
+  {
+    fakerMethod: 'database',
+    output: ['column', 'engine', 'structure']
+  },
+  {
+    fakerMethod: 'datatype',
+    output: ['string', 'uuid', 'binary']
+  },
+  {
     fakerMethod: 'date',
     output: ['past', 'future', 'calendar']
-},{
+  },
+  {
     fakerMethod: 'finance',
     output: ['accountName', 'mask', 'investors']
-},{
+  },
+  {
+    fakerMethod: 'git',
+    output: ['commitMessage', 'shortSha', 'tag']
+  },
+  {
+    fakerMethod: 'hacker',
+    output: ['noun', 'phrase', 'exploits']
+  },
+  {
+    fakerMethod: 'image',
+    output: ['avatar', 'sports', 'filter']
+  },
+  {
     fakerMethod: 'internet',
     output: ['avatar', 'userName', 'protocols']
-},{
+  },
+  {
+    fakerMethod: 'lorem',
+    output: ['words', 'sentence', 'ipsum']
+  },
+  {
+    fakerMethod: 'music',
+    output: ['genre', 'songName', 'lyrics']
+  },
+  {
     fakerMethod: 'name',
     output: ['firstName', 'prefix', 'surname']
-},{
+  },
+  {
     fakerMethod: 'phone',
     output: ['phoneNumber', 'phoneNumberFormat', 'emergency']
-},{
+  },
+  {
     fakerMethod: 'random',
-    output: ['uuid', 'word', 'checks']
-},{
-    fakerMethod: 'hacker',
-    output: ['noun', 'phrase', 'exploits']
-},{
-    fakerMethod: 'helpers',
-    output: ['userCard', 'createTransaction', 'halp']
```

Comment:
We sadly cant test the `helpers` module with your current test setup, as [all methods](https://fakerjs.dev/api/helpers.html) require parameters. Should I write an additional standalone test to cover that module itself?